### ISSUE TITLE
Ajustes solicitados na revisão do PR #164

### ISSUE THREAD

**Issue by rafaelpadilla** [rafaelpadilla/3W#17]
Durante a entrega da v.2.0.0, recebemos a revisão do PR #164 (detalhes disponíveis [neste PDF](https://github.com/user-attachments/files/22892425/251009Avaliacao3WToolkit200.pdf)) e foram identificados os seguintes pontos que requerem atenção e ajustes:

- [x] Imagens não exibidas no toolkit_2.0.0/README.md
Links quebrados:
- docs/figures/3w_top_view.png
- docs/figures/3w_data_loader.png
- docs/figures/3w_model_dev.png
- docs/figures/3w_assessment.png
- docs/figures/3w_toolkit_uml.png
- docs/figures/docker-logo-blue.png

----------

- [x] Uso do Docker
- Requer permissões administrativas.
- Sugerido destacar que o Toolkit Setup Docker é alternativo e não a opção principal de instalação.

----------

- [x] Ambiente virtual e instalação
- Conflito entre versões de Python e dependências ao usar environment.yml.
- Instalação via pip install ThreeWToolkit requer Python=3.10.11.
- Dependências listadas em toolkit_2.0.0/pyproject.toml.
@rafaelpadilla sugere 2 alternativas de instalacao:
- *Ambiente conda:* Sobrescrever o arquivo `environment.yml` existente no repositorio oficial com as dependencias do nosso projeto, permitindo que o usuario instale via `conda env create -f environment.yml`
- *Pip:* Colocarmos o nosso `pyproject.toml` na raiz do projeto para que o usuario consiga instalar via `pip install -e .`

----------

- [x] Arquivos .gitignore duplicados: Um na raiz do repositório e outro em 3W/toolkit_2.0.0/.gitignore.

----------

Foram relatatos erros em varios notebooks:
```
- Erro de import no notebooks/assessment_examples.ipynb
- Função ConfigAssessmentVisualization não encontrada em ThreeWToolkit/assessment/assessment_visualizations.py.
- Erro de import no notebooks/common_preprocessing.ipynb
- Função DatasetConfig não encontrada em ThreeWToolkit/dataset/parquet_dataset.py.
- Erro SSL no notebooks/dataset_examples.ipynb. Python não confia no certificado HTTPS. Possivelmente ocorre apenas em redes corporativas (ex: Petrobras). Precisa ser investigado e documentado.
- Erro de import no notebooks/feature_extraction_examples.ipynb
- Função DatasetConfig não encontrada em ThreeWToolkit/dataset/parquet_dataset.py.
- Erro no notebooks/load_model_examples.ipynb. Formato .xyz não suportado. BytesIO não aceito (função espera caminho de arquivo). Pickle inválido (“n”) – arquivo corrompido ou aberto em modo texto. PyTorch ≥ 2.6: precisa usar weights_only=False apenas se o arquivo for confiável.
- Erro no notebooks/mlp_timeseries_example.ipynb.  Os arquivos .parquet esperados não existem em ../../data/raw/
- Erro no notebooks/plot_series_examples.ipynb. Quantidade de valores excede o tamanho do índice.
- Erro no notebooks/report_generation_examples.ipynb. Instanciação de ReportGeneration sem os argumentos obrigatórios predictions e calculated_metrics.
- Erro no notebooks/sklearn_models_examples.ipynb. Função make_classification não importada.
```
- [x] Devemos rodar TODOS nossos notebooks para garantir que nao existem problemas.

**Comment by czewski** [comment]
Items that still need attention: 
- Ambiente virtual e instalação
- Uso do Docker


And from the #18 updates: 
1. We should not need "_sys.path.append(os.path.abspath(os.path.join(os.getcwd(), "../../")))_", as we are setting a requirement for the user to install the toolkit properly. 

2. Notebooks to update: (Use 3W dataset, Add more comments)- load_model_examples.ipynb --> Update documentation
- **report_generation_examples**.ipynb. --> It has a problem related to not founding the correct template for the HTML report. Should be updated. 
- **train_test_split_examples**.ipynb --> Suggestion: Remove notebook, as the Toolkit does not have this function anymore (just a method inside the trainer). 
	- BaseTimeSeriesHoldout has nothing. 
- **feature_extraction_examples**.ipynb --> Exponentially Weighted Statistical Feature Extraction takes more than 40min to run, something may be wrong. 
- **common_preprocessing**.ipynb - I made half of this notebook work, but somebody have to update it, adding relevant comments. 
	- The ParquetDataset class does not have the transform method anymore, is there a way to still use default_data_normalization, default_data_processing?
	- I saw that we have Normalize() method, maybe we should remove the methods mentioned above?
